examples/deploy-krops.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script demonstrates how to setup a nix-bitcoin node with krops.
+# Running this script leaves no traces on your host system.
+
+# This demo is a template for your own experiments.
+# Feel free to modify or to run nix-shell and execute individual statements of this
+# script in the interactive shell.
+
+# MAKE SURE TO REPLACE the SSH identity file if you use this script for
+# anything serious.
+
+if [[ ! -v IN_NIX_SHELL ]]; then
+    echo "Running script in nix shell env..."
+    cd "${BASH_SOURCE[0]%/*}"
+    exec nix-shell --run "./${BASH_SOURCE[0]##*/} $*"
+fi
+
+cd "${BASH_SOURCE[0]%/*}"
+
+tmpDir=/tmp/nix-bitcoin-krops
+mkdir -p $tmpDir
+
+# Cleanup on exit
+cleanup() {
+    set +eu
+    kill -9 $qemuPID
+    rm -rf $tmpDir
+}
+trap "cleanup" EXIT
+
+identityFile=qemu-vm/id-vm
+chmod 0600 $identityFile
+
+echo "Building VM"
+nix-build --out-link $tmpDir/vm - <<EOF
+(import <nixpkgs/nixos> {
+  configuration = {
+    virtualisation.graphics = false;
+    virtualisation.diskSize = 8192;
```

Comment:
`qemu-vm.nix` makes the Nix store in the VM writable by layering an overlay filesystem on top of the host's Nix store. This overlay has a very limited size when using Tmpfs. Instead we should use `virtualisation.writableStoreUseTmpfs = false;` which makes the VM write to its own filesystem (8192MB) so we can use the writableStore with more featured deployments.